rfcs/text/0014-extend-file-pe.md

```diff
@@ -77,17 +78,217 @@ Stage 1: Provide a high-level description of example sources of data. This does
 
 <!--
 Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting.
+https[://]c4a08b8258a848efb518dcea18a1d11e[.]us-central1[.]gcp[.]cloud[.]es[.]io/goto/51d883b52d4ad99d9200c1b25cf1b5c9
```

Comment:
Is this something we want to add to the comments here, or is it just something that was left in unintentionally?

Reply:
Unintentionally. Removed.